core/modules/filter/filter.module

```diff
@@ -2003,6 +2003,16 @@ function filter_filter_info() {
     'process callback' => '_filter_image_align',
     'weight' => 5,
   );
+  $filters['filter_image_style'] = array(
+    'title' => t('Image styles in content'),
+    'description' => t('Apply image styles to images in content.'),
+    'process callback' => '_filter_image_style_process',
+    'settings callback' => '_filter_image_style_settings',
+    'default settings' => array(
+      'image_styles' => array(),
+    ),
+    'tips callback' => '_filter_image_style_tips',
+  );
```

Comment:
I think the tests might be failing because Image module is not actually required, so this may need to be wrapped in a `if module_exists('image') {` condition. Or... maybe the entire file and functionality it should be moved to image module?